### PR TITLE
cmds/core/watchdog: add tests

### PR DESCRIPTION
**Pull request by binjip978**

### CODE REVIEW

File: cmds/core/watchdog/watchdog_linux_test.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+import (
+	"errors"
+	"os"
+	"testing"
+)
+
+// TestRun test watchdog cli against a regular file, most tests expected to
+// return an error, except of "keepalive"
+func TestRun(t *testing.T) {
+	dev, err := os.CreateTemp("", "")
```
Comment (rminnich) [inline cmds/core/watchdog/watchdog_linux_test.go:16]: could you use t.TempDir() here? Then you would not need that defer below, etc.

Comment (binjip978) [reply cmds/core/watchdog/watchdog_linux_test.go]: done

File: cmds/core/watchdog/watchdog_linux.go [hunk 2]
```diff
@@ -62,71 +62,76 @@ func runCommand() error {
...
+			return errUsage
 		}
 		i, err := wd.TimeLeft()
 		if err != nil {
 			return err
 		}
 		fmt.Println(i)
 	default:
-		return fmt.Errorf("unrecognized command: %q", flag.Arg(0))
+		return fmt.Errorf("unrecognized command: %q", args[0])
```
Comment (rminnich) [inline cmds/core/watchdog/watchdog_linux.go:123]: There is so much duplicated code here I keep thinking this could be done with a table. Maybe later.

**Review by rminnich — APPROVED** [review]

File: cmds/core/watchdog/watchdog_linux.go (hunk 2 above)
Comment (binjip978) [reply cmds/core/watchdog/watchdog_linux.go:123]: yeah, but let's add some tests firsts :)

**Pull request merged**